packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -18,14 +18,45 @@ import 'scroll_metrics.dart';
 import 'scroll_notification.dart';
 import 'scroll_physics.dart';
 
-// ## Subclassing ScrollPosition
-//
-//  * Describe how to impelement [absorb]
-//     - May need to start an idle activity
-//     - May need to update the activity's idea of what the delegate is
-//     - etc
-//  * Need to call [didUpdateScrollDirection] when changing [userScrollDirection]
+/// Determines which portion of the content is visible in a scroll view.
+///
+/// The [pixels] value determines the scroll offset that the scroll view uses to
+/// select which part of its content to display. As the user scrolls the
+/// viewport, this value changes, which changes the content that is displayed.
+///
+/// The [ScrollPosition] applies [physics] to scrolling, and stores the
+/// [minScrollExtent] and [maxScrollExtent].
+///
+/// Scrolling is controlled by the current [activity], which is set by
+/// [beginActivity]. [ScrollPosition] itself does not start any activities.
+/// Instead, concrete subclasses, such as [ScrollPositionWithSingleContext]
+/// typically start activities in response to user input or instructions from a
+/// [ScrollController].
+///
+/// This object notifies its listeners when [pixels] changes.
```

Comment:
this leaves open the question of how; maybe "This object is a Listenable that notifies..."?